com.github.debauchee.barrier.appdata.xml

```diff
@@ -0,0 +1,104 @@
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.debauchee.barrier</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Barrier</name>
+  <!-- From the Github repo -->
+  <summary>Open-source KVM software</summary>
+  <description>
+  What is it?
+
+  Barrier is KVM software forked from Symless's synergy 1.9 codebase. Synergy was a commercialized reimplementation of the original CosmoSynergy written by Chris Schoeneman.
+
+  What's different?
+
+  Whereas synergy has moved beyond its goals from the 1.x era, Barrier aims to maintain that simplicity. Barrier will let you use your keyboard and mouse from machine A to control machine B (or more). It's that simple.
+
+  Project goals
+
+  Hassle-free reliability. We are users, too. Barrier was created so that we could solve the issues we had with synergy and then share these fixes with other users.
+
+  Compatibility. We use more than one operating system and you probably do, too. Windows, OSX, Linux, FreeBSD... Barrier should "just work". We will also have our eye on Wayland when the time comes.
+
+  Communication. Everything we do is in the open. Our issue tracker will let you see if others are having the same problem you're having and will allow you to add additional information. You will also be able to see when progress is made and how the issue gets resolved.
+
+  Contact &amp; support
+
+  Please be aware that the only way to draw our attention to a bug is to create a new PR in the issue tracker. Write a clear, concise, detailed report and you will get a clear, concise, detailed response. Priority is always given to issues that affect a wider range of users.
+
+  For short and simple questions or to just say hello find us on the Freenode IRC network in the #barrier channel.
+
+  Contributions
+
+  At this time we are looking for developers to help fix the issues found in the issue tracker. Submit pull requests once you've polished up your patch and we'll review and possibly merge it.
+  </description>
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apa.html -->
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apas02.html -->
+  <categories>
```

Comment:
Put the categories on the desktop file only. They are grabbed from there once you run appstream-compose.

Reply:
Ah, thanks, Haven't done this stuff in a while.